src/likesAPI.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+import http from './http.js';
+
+// links
+const appKey = 'iTwOeYcQW4Syd1iyW4ip';
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+const likesLink = `/apps/${appKey}/likes/`;
+
+// Get Likes Count
+const getLikes = async () => {
+  const result = await http.get(`${baseUrl}${likesLink}`);
+  console.log(result);
+  return result;
+};
+
+// add likes
+function addLike(itemID) {
+  console.log('ti id einai edw mesa addLike', itemID);
+  const likeBody = {
+    item_id: itemID,
+  };
+
+  http.post(`${baseUrl}${likesLink}`, likeBody)
+    .then((data) => {
+      console.log(data);
+      getLikes();
+    })
+    .catch((err) => console.log(err));
+}
```

Comment:
- [ ] Like count seems like getting updated on the page now, but the change of the like count doesn't get updated correctly when I refresh the page. Kindly fix this issue.

Reply:
It is working perfectly for me now I don't know what you are talking about.

Reply:
Like when we click on like button the likes number will increase but after that when we refresh the page the likes number doesn't show the correct updated number of like. That is the issue.

Reply:
If you took the time as a TSE to really review the code or even take the time and watch the video, which has 0 Views, then you would be able to see that when you click the like btn, you post a like to the API and the counter to show on the page. Counter just adds on a click for not waiting for the slow API. On the API side sometimes when you spam you get a false post, so no update to the API and when you update you see that result. But anyway I didn't get approved on time so I have to repeat the Module.